### PR TITLE
Feature adding disqus section

### PR DESCRIPTION
**Pull request by Melissa-Martinez-XL**
## Ticket

* [Add a DISQUS comment section in the blog’s page](https://www.notion.so/xmartlabs/Add-a-DISQUS-comment-section-in-the-blog-s-page-07ce067986c846bf88c3fa166b3134ab?pvs=4)

## Type of change

* [ ] Fix
* [X] Story
* [ ] Chore

## Description of the change

I added the comment section to the blog page using Disqus.

## Screenshot/Execution

Captura desde 2023-04-05 15-48-04

https://user-images.githubusercontent.com/107500715/230176681-48369f43-b95c-4405-a539-ef89d9b86f66.mp4

## Related PRs
Links to the related PRs

### CODE REVIEW

**Comment by Melissa-Martinez-XL** [comment]
@gpicart and @lauta2912 I'd like to try the comments section, so if you don't mind, I invite you to leave a comment in Disqus in the [deploy preview link.](https://deploy-preview-47--xl-blog.netlify.app/) It may not allow it, but we can try.

File: src/css/blog-post.module.scss [hunk 1]
```diff
@@ -307,12 +303,24 @@
   }
 }
 
+// DISQUS
+
+.disqusSection {
+  padding: 0 6rem 0 6rem;
```
Comment (lauta2912) [inline src/css/blog-post.module.scss:309]:
```suggestion
  padding: 0 6rem;
```

**Comment by lauta2912** [comment]
> @gpicart and @lauta2912 I'd like to try the comments section, so if you don't mind, I invite you to leave a comment in Disqus in the [deploy preview link.](https://deploy-preview-47--xl-blog.netlify.app/) It may not allow it, but we can try.

I get this
image

File: src/templates/post.js [hunk 2]
```diff
@@ -130,7 +131,13 @@ const BlogPost = ({ data }) => {
       return null;
     }
     return null;
-  }; 
+  };
+
+  let disqusConfig = {
```
[the remark this answers is not in the document]
Comment (Melissa-Martinez-XL) [reply src/templates/post.js:136]: Done

File: src/css/blog-post.module.scss (hunk 1 above)
Comment (Melissa-Martinez-XL) [reply src/css/blog-post.module.scss:309]: Done

File: src/templates/post.js [hunk 3]
```diff
@@ -170,6 +177,12 @@ const BlogPost = ({ data }) => {
         <SocialElement className={classnames(styles.socialBottom, styles.blogIcons)} links={shareXlProfileLinks} />
       </div>
       <MoreBlogsSection data={data} refMoreFrom={refMoreFrom} title={categoryBlog.displayName} />
+      <div className={styles.disqusSection}>
+        <h3 className={styles.disqusTitle}>Comments:</h3>
+        <div id="disqus_thread">
+          <Disqus config={disqusConfig} />
```
[the remark this answers is not in the document]
Comment (Melissa-Martinez-XL) [reply src/templates/post.js:193]: Done

**Review by lauta2912 — APPROVED** [review]

**Pull request merged**